build.yaml

```diff
@@ -1,8 +1,8 @@
 nodejs:
   - "0.10"
   - "0.12"
-  - "iojs-3"
-  - "4.0"
+  - "4.3"
```

Comment:
I've tried to use v4 and v5 (without minor) on build.yaml file https://github.com/datastax/nodejs-driver/pull/136/commits/a5fc268804d098db0059c13bc7b67560d4c5b11f but it seems that needs settings update on the Jenkins side. I've restored it to use major/minor runtime versions.